src/services/user/services/SkipRegistration.js

```diff
@@ -109,6 +98,35 @@ class SkipRegistrationService {
 		}
 	}
 
+	/**
+	 * POST /users/{id}/skipregistration
+	 * skips the registration of another user, making the necessary changes as if the user went through registration.
+	 * Only works on users that have an importHash.
+	 * @param {Object} data should contain consents, a password, and if the target is a student a birthday.
+	 * @param {Object} params accepts no params besides the ones automatically set by feathers.
+	 * @returns 201 - success
+	 * @throws {BadRequest} if data is not set correctly
+	 * @throws {Forbidden} if caller does not have permission to skip the target users registration.
+	 * @memberof SkipRegistrationService
+	 */
+	async create(data, params) {
```

Comment:
In dem Services wird eine userId via route geforced. Das aber wiederspricht der Logik das man jetzt die userIds via data mit gibt. Ich würde vorschlagen das hier die id aus der Route entfernt wird, und die Aufrufe im alten/neuen Client und intern auf data umgebogen werden.
Ansonsten muss man irgendwas in die route bei id rein schreiben um dann die bulk operation ausführen zu können.

Reply:
du kannst entweder über /users/:userId/skipregistration gehen und gibst die id in der route,
oder über /users/skipregistration gehen und die id via data mitgeben.

Reply:
ok